tests/integration/ceph_upgrade_test.go

```diff
@@ -146,23 +146,23 @@ func (s *UpgradeSuite) testUpgrade(useHelm bool, initialCephVersion v1.CephVersi
 	// do not need retry b/c the OBC controller runs parallel to Rook-Ceph orchestration
 	assert.True(s.T(), s.helper.BucketClient.CheckOBC(obcName, "bound"))
 
-	logger.Infof("Verified upgrade from %s to master", installer.Version1_7)
+	logger.Infof("Verified upgrade from %s to master", installer.Version1_8)
 
 	// SKIP the Ceph version upgrades for the helm test
 	if s.settings.UseHelm {
 		return
 	}
 
 	//
-	// Upgrade from octopus to pacific
+	// Upgrade from pacific to quincy
 	//
-	logger.Infof("*** UPGRADING CEPH FROM OCTOPUS TO PACIFIC ***")
-	s.gatherLogs(s.settings.OperatorNamespace, "_before_pacific_upgrade")
-	s.upgradeCephVersion(installer.PacificVersion.Image, numOSDs)
+	logger.Infof("*** UPGRADING CEPH FROM PACIFIC TO QUINCY ***")
+	s.gatherLogs(s.settings.OperatorNamespace, "_before_quincy_upgrade")
+	s.upgradeCephVersion(installer.QuincyVersion.Image, numOSDs)
 	// Verify reading and writing to the test clients
-	newFile = "post-pacific-upgrade-file"
+	newFile = "post-quincy-upgrade-file"
 	s.verifyFilesAfterUpgrade(newFile, rbdFilesToRead, cephfsFilesToRead)
-	logger.Infof("Verified upgrade from octopus to pacific")
+	logger.Infof("Verified upgrade from pacific to quincy")
```

Comment:
Do we want to instead make a copy and test octopus->pacific->quincy?

Reply:
We probably should just to be sure. My hesitancy is just that historically the upgrade from one previous version of ceph was sufficient to catch rook upgrade issues, rather than upgrading through both major versions.